internal/configuration/schema/spnego.go

```diff
@@ -0,0 +1,8 @@
+package schema
+
+type SPNEGO struct {
+	Disable   bool   `koanf:"disable" json:"disable" jsonschema:"default=false,title=Disable" jsonschema_description:"Disables the SPNEGO authentication functionality."`
```

Comment:
```suggestion
	Enabled   bool   `koanf:"enabled" json:"enabled" jsonschema:"default=false,title=Enabled" jsonschema_description:"Enables the SPNEGO authentication functionality."`
```

Should be explicitly opt-in. Adjust this and the related code/docs.